src/main/java/com/thebois/models/world/World.java

```diff
@@ -50,4 +57,56 @@ public Object find() {
         return copy;
     }
 
+    /**
+     * Returns the structures in an arraylist as the interface IStructures.
+     *
+     * @return The list to be returned.
+     */
+    public ArrayList<IStructure> getStructures() {
+        final ArrayList<IStructure> copy = new ArrayList<>();
+        for (final IStructure[] matrix : structuresMatrix) {
+            for (final IStructure structure : matrix) {
+                if (structure != null) {
```

Comment:
We maybe should've discussed this at the meeting today, but we should avoid null at all costs. Instead of using `null` to represent "no structure here", try using an `Option<IStructure>`. Or maybe creating an implementation of `IStructure`, `NoStructure` that is used just to represent empty tiles.